ui/web-ui-nginx-site.conf

```diff
@@ -50,6 +50,7 @@ http {
 
       location @rewrites {
         rewrite ^\/docs.*$ /docs.html last;
+        rewrite ^\/__version__$ /app/version.json last;
         rewrite ^.+$ /index.html last;
```

Comment:
What should the `__lbheartbeat__` and `__heartbeat__`  endpoints redirect to? Unlike `__version__`, they don't point to a json file but rather are endpoints exposed by each service.

Reply:
It sounds like they should just always return a 200 status and an empty JSON body.  We can go back later and check on DB connectivity, etc.

Reply:
Ah great. The current state of the PR does just that.

Reply:
I think this will need `__(lb)heartbeat__` too?